test/meta-colony.js

```diff
@@ -45,6 +47,11 @@ contract("Meta Colony", accounts => {
     const metaColonyAddress = await colonyNetwork.getMetaColony();
     metaColony = await IColony.at(metaColonyAddress);
 
+    const reputationMiningCycle = await ReputationMiningCycle.new();
+    const reputationMiningCycleRespond = await ReputationMiningCycleRespond.new();
+    const reputationMiningCycleResolver = await Resolver.new();
+    await setupReputationVersionResolver(reputationMiningCycle, reputationMiningCycleRespond, reputationMiningCycleResolver, colonyNetwork);
```

Comment:
Same comment as for the `colony` test setup, this is already done in migrations.